data/core/units/saurians/Prophet.cfg

```diff
@@ -0,0 +1,91 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Prophet
+    name= _ "Saurian Prophet"
+    race=lizard
+    gender=male
+    image="units/saurians/prophet/prophet.png"
+    profile="portraits/saurians/augur.png"
+    hitpoints=43
+    movement_type=lizard
+    movement=6
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=51
```

Comment:
It feels like this cost is kind of high, or maybe Seer's cost is kind of low in comparison.

Reply:
The prices are right, this is their respective power level.

Reply:
The prices and other statistics were discussed with @Hejnewar beforehand. Do you propose any other values?

Reply:
Well that's fine Hejne's prices are fine. But I would still think them being weaker in melee and stronger in ranged is better. But I'll discuss with him, I'm flexible.

Reply:
I meant the mages statistics, I think you referencing the Spearthrower stats. Do you also want to raise the damage of Seer/Prophet?

Reply:
For the record, Seer and Prophet were checked by Hejne before, along with their costs. Spearthrower line was more of a suggestion on my side that still needs to be verified, since originally the unit was a more defensive mixed fighter (as per current stats on #6905 PR).

Reply:
Yes, I meant the spearthrower, these are fine in stats.